apiserver/Makefile

```diff
@@ -32,6 +35,10 @@ jupyter_notebook_config.py:
 	    -e "s,@keyfile@,$(GSA_KEY_FILE),g" \
             < jupyter_notebook_config.py.in > $@
 
+hailjwt: ../hailjwt/setup.py $(shell find ../hailjwt/hailjwt -iname \*.py)
+	cd ../ && rsync --relative $(?:../hailjwt/%=hailjwt/./%) apiserver/hailjwt
```

Comment:
Two comments:

This is only used by phony rules (build), I think it should also be phony.  Don't attempt to track dependencies.  Docker will do that in the build.  rsync is super fast.

I find just `rsync -r --delete ../hailjwt/setup.py ../hailjwt/hailjwt hailjwt` to be much clearer than the `--relative` business.  (I also don't see how this works without `-r`.)  `--delete` is necessary in case files get deleted from hailjwt.

Reply:
--recursive isn't necessary because the files are explicitly listed. I switched to your suggestion.

Reply:
Ah, I got confused between rsync syntax, shell syntax and make syntax.